### DISCUSSION TITLE
How to store the objective function and constraints in a database?

### DISCUSSION THREAD

**Discussion by dassouki** [coin-or/pulp#350] [Q&A] [answered]
Hi folks, I'm building an app that allows a user to save their models as different "Scenarios" where they can use a web interface to manually adjust the objective function and some of the constraints. The whole app is built and functioning fine, except for figuring out how to store the OBJ/Cons in the db/table and how to show them back to the user.

For the sake of example, let's use the [Two stage production program](https://coin-or.github.io/pulp/CaseStudies/a_two_stage_production_planning_problem.html) found in the docs. 

**My question becomes, how would you design the table to store these items for access by the user?**

**Comment by pchtsp** [comment, marked as answer]
Maybe check this part of the docs: https://coin-or.github.io/pulp/guides/how_to_export_models.html

**Reply by dassouki** [reply]
Thanks @pchtsp I'm assuming I can also use the same logic to import it back somehow and solve it or allow the user to do adjustments and edit of the constraints and objective function?
